reV/generation/base.py

```diff
@@ -34,6 +34,71 @@ class BaseGen(ABC):
     # Mapping of reV generation / econ outputs to scale factors and units.
     OUT_ATTRS = {}
 
+    # Mapping of reV econ outputs to scale factors and units.
+    # Type is scalar or array and corresponds to the SAM single-site output
+    # This is the OUT_ATTRS class attr for Econ but should also be accessible
+    # to rev generation
+    ECON_ATTRS = {'other': {'scale_factor': 1, 'units': 'unknown',
```

Comment:
Not essential for this PR, but I wonder if we should think about moving teh ECON and GEN attrs to JSONs and then adding in kwargs. If the kwargs are None the defaults are imported. I'm just wondering if anyone might want to change the attrs...

Reply:
Yeah i agree... Remind me, if we put json files in sub directories in the reV module directories will they get copied over when pip installed?

Reply:
yup, just like the SAM default .csvs, we just have to add to this list:
https://github.com/NREL/reV/blob/ce5ef74aabd6398895e03017143d6aa560fd013c/setup.py#L78

Reply:
Done, can you check this commit? I figure we can add something to modify the attributes on the fly later. 

https://github.com/NREL/reV/pull/253/commits/9c3e4f9db0156f8f4a60a0f40d94d3873ff0d99e

Reply:
Now this commit (after rebase): https://github.com/NREL/reV/pull/253/commits/bf05da6c1fc06b444af97bad2b6481d5e61a7e69